src/components/CreditCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+function CreditCard(props) {
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props;
+
+    let logoSrc;
+    let logoAlt;
+
+    if (type === "Visa") {
+        logoSrc = "/src/assets/images/visa.png"
```

Comment:
Your images are not being displayed. Remember that you need to import your images if you want to use it.